package.json

```
@@ -10,6 +10,7 @@
     "raster"
   ],
   "main": "dist-node/geotiff.js",
+  "types": "dist-node/geotiff.d.ts",
```

Since `src` is included in the build, I think this can be simplified:

```suggestion
  "types": "src/geotiff.d.ts",
```

---

Good catch, that simplifies things

---

changed